configuration/configuration.go

```diff
@@ -31,15 +31,17 @@ type API struct {
 }
 
 type DB struct {
-	URL      string `insconfig:"postgres://postgres@localhost/postgres?sslmode=disable| Path to postgres db"`
-	PoolSize int    `insconfig:"100| Maximum number of socket connections"`
+	URL string `insconfig:"postgres://postgres@localhost/postgres?sslmode=disable| Path to postgres db"`
+	// for nondocker testing also it may be  `postgres://postgres:secret@localhost:5432/postgres?sslmode=disable`
```

Comment:
I think this comment should be above line N34

Reply:
yes. may be.